src/OpenCensus.Collector.Dependencies/DependenciesCollectorOptions.cs

```diff
@@ -16,10 +16,30 @@
 
 namespace OpenCensus.Collector.Dependencies
 {
+    using System;
+    using OpenCensus.Trace;
+    using OpenCensus.Trace.Sampler;
+
     /// <summary>
     /// Options for dependencies collector.
     /// </summary>
     public class DependenciesCollectorOptions
     {
+        private static Func<Uri, ISampler> defaultSampler = (uri) => { return uri.ToString().Contains("zipkin.azurewebsites.net") ? Samplers.NeverSample : Samplers.AlwaysSample; };
```

Comment:
I'd suggest to return `Samplers.NeverSample` in case of `zipkin` url and `null` otherwise. Basically only return something meaningful when you know how to handle this specific span.